lib/command/diff.php

```diff
@@ -131,11 +152,131 @@ private function createDiff(array $schema, array $tables, $allowEmpty)
         foreach ($schema as $tableName => $table) {
             $diff->dropTable($tableName);
         }
+    }
 
-        if ($diff->isEmpty() && !$allowEmpty) {
-            return null;
+    /**
+     * @param rex_sql_table[]       $tables
+     * @param rex_ydeploy_diff_file $diff
+     */
+    private function handleFixtures(array $tables, rex_ydeploy_diff_file $diff)
+    {
+        $fixtureTableNames = [
+            'action',
+            'markitup_profiles',
+            'media_manager_type',
+            'media_manager_type_effect',
+            'metainfo_field',
+            'metainfo_type',
+            'module',
+            'module_action',
+            'redactor2_profiles',
+            'template',
```

Comment:
Können wir den text nicht auch mehrzeilig in den fixtures haben sodass wir die gleichen vorteile haben wie wenn es plain files sind (wie vom developer-addon)?